src/gtest/test_rpc.cpp

```diff
@@ -10,33 +10,6 @@
 #include "streams.h"
 #include "utilstrencodings.h"
 
-TEST(rpc, GetDifficultyTestnetRules) {
-    SelectParams(CBaseChainParams::TESTNET);
```

Comment:
Oh, is there no mainnet test corresponding to this? I just meant have a test that would fail without this commit.